propagators/aws/xray/xrayidgenerator/aws_xray_idgenerator.go

```diff
@@ -0,0 +1,73 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
aws and xray are already part of the file path.  Should this filed just be named `idgenerator.go`?